jobs/admin.py

```diff
@@ -1,6 +1,19 @@
 from django.contrib import admin
-
 from jobs.models import Job
 
 
-admin.site.register(Job)
+@admin.register(Job)
+class JobAdmin(admin.ModelAdmin):
+
+    fieldsets = (
+        ('Temel Bilgiler', {
```

Comment:
Aslında karar veremedim. İngilizce, Türkçe karışıktı kodlarda bu tarz alanlar. verbose_name'ler genelde Türkçe yazıldığı için bende bu kısmı Türkçe yazmıştım.